### ISSUE TITLE
GM control panel for PCs

### ISSUE THREAD

**Issue by eggdropsoap** [eggdropsoap/questworlds-fvtt#32]
A window for the GM with an overview of the player-characters. Should list basic information, click to open full PC sheet, and controls for managing XP and Story Points.

**Comment by eggdropsoap** [comment]
No longer necessary after #35, but this is still a nice-to-have.
